cavy.js

```diff
@@ -47,13 +49,23 @@ program
   .command('run-ios')
   .description('Run cavy spec on an ios simulator or device')
   .option('-f, --file <file>', 'App entry file')
+  .option(
+    '-nb, --nobuild',
```

Comment:
The short hand flag for nobuild doesn't work at the moment - I believe it needs to be a single character https://github.com/tj/commander.js/#options

Reply:
Ahhh.... thanks! I actually just noticed yesterday that it wasn't working but hadn't figured out why.

Reply:
Switched to `-s, --skipbuild`

Reply:
Alternate possible option name that I just thought of would be `-p, --passive`... but `skipbuild` is perhaps more clear/explicit

Reply:
Nice! I prefer -s and --skipbuild👌